costar_robot/costar_robot_manager/src/costar_robot/ur_driver.py

```diff
@@ -154,35 +154,4 @@ def send_cart(self,q,acceleration,velocity):
         #  cmd = list(T.p) + [angle*axis[0],angle*axis[1],angle*axis[2]]
         #  self.ur.movel(cmd,wait=True,acc=acceleration,vel=velocity)
 
-    def handle_tick(self):
-
-        # send out the joint states
-        #self.current_joint_positions = self.ur.getj_all()
-        #self.q0 = np.array(self.current_joint_positions[0])
-        #self.q_v0 = np.array(self.current_joint_positions[1])
-        #self.q_a0 = np.array(self.current_joint_positions[2])
-        #self.js_publisher.publish(JointState(
-        #  header=Header(stamp=rospy.Time.now()),
```

Comment:
i mightve deleted it when we were working together though...  the deleted code does not seem to do anything.